### PR TITLE
WIP: Add up and down command/scripts and make portainer optional

### PR DESCRIPTION
**Pull request by chregu**
Needs #1 first merged, therefore WIP for now

- add `scripts/pontsun up` and `scripts/pontsun down`
- makes portainer startup optional via .env
- Manual `docker-compose up` still works

### CODE REVIEW

**Comment by Aerzas** [comment]
I'm not sure if we should abstract docker-compose which is already an abstraction of docker. Templates and commands are quite straightforward.

Portainer is already marked as `restart: unless-stopped` which would prevent it from starting if stopped.

**Comment by chregu** [comment]
Yeah, for portainer maybe not. But there's more services in future to be added. A dnsmasq service eg. is not needed on all setups and also the ssh-agent service may only be needed on OS X (and maybe windows). So I'd like to have a better scriptable "up" command.

But sure, the portainer thing is quite optional. And I'd also like to keep this "up" command optional, more of a helper for getting pontsun getting started faster for non-experts with some checks (like the .env check)

And as long as "docker-compose up" still works, I don't see how such a script hurts.

File: scripts/up.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+if [[ "$PONTSUN_PORTAINER" != "no" ]]; then
+  SERVICES="$SERVICES portainer"
+fi
+
+echo "Starting pontsun services: $SERVICES"
+
+docker-compose up -d $SERVICES
+
+if ! ping -c 1 -t 2 traefik.$PROJECT_DOMAIN  > /dev/null 2>&1 ; then
+    echo "!!!! traefik.$PROJECT_DOMAIN could not be resolved. You should fix this !!!!"
```
Comment (OdyX) [inline scripts/up.sh]: [NITPICK] Please use `>&2` at the line start, to output to stderr, not stdout.

**Comment by chregu** [comment]
Also that I can do  
```
$PATH_TO_PONTSUN/scripts/pontsun up
```
 from any place and not need to do
```
cd $PATH_TO_PONTSUN/containers && docker-compose up -d
cd $BACK_TO_WHERE_I_WAS
```
which is much easier to remember, type, use-in-other-scripts

File: scripts/up.sh (hunk 1 above)
Comment (chregu) [reply scripts/up.sh]: Sure, changed.

**Comment by Aerzas** [comment]
I mean we could achieve the same results with multiple docker-compose.yml files like `docker-compose -f docker-compose.yml -f docker-compose.dnsmasq.yml up -d` so options are explicit and yet we don't create an abstraction layer.

My opinion is that we should ease the setup of local dev environment without replacing docker or docker-compose or maybe in a separated project.

Moreover, we'll probably execute this command once or twice to mount it once and it will remount at every startup automatically.

**Comment by chregu** [comment]
Ah, didn't know you can have more than one -f, good then.

We seem not to agree, what this project should include and what not. So let's create a pontsun-cli project and move what my vision of this "cli" is there and leave this here pure.

**Comment by chregu** [comment]
Obsolete. That stuff will go into a new project.

**Pull request closed without merging**